internal/envoy/auth/oauth2_filter.go

```diff
@@ -119,7 +119,7 @@ func NewFilterHTTPOAuth2(oauth2Options *options.OAuth2, args *ParseAuthArguments
 		clientSecret = string(secret.Data["client_secret"])
 	}
 
-	logger.Info("auth.NewFilterHTTPOAuth2: set client_secret", "client_secret", clientSecret)
+	_ = clientSecret
```

Comment:
What is this for?

Reply:
Prevent the `client_secret` from being printed. I asked @jasmingacic to make that change.